<ui version="4.0" >
 <class>MvdMainInfoPage</class>
 <widget class="QWidget" name="MvdMainInfoPage" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <item row="0" column="0" >
    <widget class="QLabel" name="labCaptionMessage" >
     <property name="text" >
      <string>&lt;b&gt;Main Information&lt;/b&gt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="labInfoMessage" >
     <property name="text" >
      <string>You must enter at least the original title or a localized title.</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0" >
    <layout class="QHBoxLayout" name="horizontalLayout_4" >
     <item>
      <layout class="QFormLayout" name="formLayout" >
       <item row="0" column="0" >
        <widget class="QLabel" name="labTitle" >
         <property name="text" >
          <string>&lt;b>Title:&lt;/b></string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="MvdClearEdit" name="title" />
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="labOriginalTitle" >
         <property name="text" >
          <string>&lt;b>Original title:&lt;/b></string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="MvdClearEdit" name="originalTitle" />
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="labVersion" >
         <property name="text" >
          <string>Edition:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="MvdClearEdit" name="version" />
       </item>
       <item row="3" column="0" >
        <widget class="QLabel" name="labStorageID" >
         <property name="text" >
          <string>Storage ID:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" >
        <widget class="MvdClearEdit" name="storageID" />
       </item>
       <item row="4" column="0" >
        <widget class="QLabel" name="labYear" >
         <property name="text" >
          <string>Release year:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" >
        <widget class="QSpinBox" name="year" >
         <property name="minimum" >
          <number>0</number>
         </property>
         <property name="maximum" >
          <number>9999</number>
         </property>
         <property name="value" >
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="5" column="0" >
        <widget class="QLabel" name="labLength" >
         <property name="text" >
          <string>Running time:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1" >
        <widget class="QSpinBox" name="lengthMinutes" >
         <property name="maximum" >
          <number>999</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_5" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>13</width>
         <height>143</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" >
       <item>
        <spacer name="verticalSpacer_5" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>18</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3" >
         <item>
          <spacer name="horizontalSpacer_6" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Preferred</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>15</width>
             <height>17</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="MvdPosterLabel" name="poster" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize" >
            <size>
             <width>46</width>
             <height>60</height>
            </size>
           </property>
           <property name="focusPolicy" >
            <enum>Qt::TabFocus</enum>
           </property>
           <property name="frameShape" >
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow" >
            <enum>QFrame::Plain</enum>
           </property>
           <property name="text" >
            <string/>
           </property>
           <property name="textInteractionFlags" >
            <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" >
         <item>
          <spacer name="horizontalSpacer_4" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Preferred</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>18</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="setPosterButton" >
           <property name="text" >
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="removePosterButton" >
           <property name="text" >
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_4" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>18</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="4" column="0" >
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>12</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0" >
    <widget class="QLabel" name="labCaptionMessage_2" >
     <property name="text" >
      <string>&lt;b&gt;Special Tags&lt;/b&gt;</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" >
    <widget class="Line" name="line_2" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="7" column="0" >
    <layout class="QHBoxLayout" name="horizontalLayout_2" >
     <item>
      <widget class="QCheckBox" name="markAsSeen" >
       <property name="text" >
        <string>Mark as seen</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="markAsSpecial" >
       <property name="text" >
        <string>Mark as special</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="markAsLoaned" >
       <property name="text" >
        <string>Mark as loaned</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="8" column="0" >
    <spacer name="verticalSpacer_2" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>12</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="0" >
    <widget class="QLabel" name="labCaptionMessage_3" >
     <property name="text" >
      <string>&lt;b&gt;Rating&lt;/b&gt;</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0" >
    <widget class="Line" name="line_3" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="11" column="0" >
    <layout class="QGridLayout" name="gridLayout_2" >
     <item row="0" column="0" >
      <widget class="MvdRatingWidget" name="ratingLabel" >
       <property name="minimumSize" >
        <size>
         <width>100</width>
         <height>20</height>
        </size>
       </property>
       <property name="focusPolicy" >
        <enum>Qt::TabFocus</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2" >
      <widget class="QLabel" name="ratingStatus" >
       <property name="focusPolicy" >
        <enum>Qt::TabFocus</enum>
       </property>
       <property name="contextMenuPolicy" >
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="text" >
        <string>Rating</string>
       </property>
       <property name="textInteractionFlags" >
        <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <spacer name="spacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>318</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="12" column="0" >
    <spacer name="verticalSpacer_3" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MvdClearEdit</class>
   <extends>QLineEdit</extends>
   <header>mvdshared/clearedit.h</header>
  </customwidget>
  <customwidget>
   <class>MvdPosterLabel</class>
   <extends>QLabel</extends>
   <header>posterlabel.h</header>
  </customwidget>
  <customwidget>
   <class>MvdRatingWidget</class>
   <extends>QLabel</extends>
   <header>ratingwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
